documentation/migrate.markdown

```diff
@@ -4,17 +4,6 @@ layout: default
 canonical: "/puppetdb/latest/migrate.html"
 ---
 
-Migrating from ActiveRecord storeconfigs
------
-
-If you're using exported resources with ActiveRecord storeconfigs, you may want to migrate your existing data to PuppetDB before connecting the master to it. This will ensure that whatever resources were being collected by the agents will still be collected, and no incorrect configuration will be applied.
-
-The existing ActiveRecord data can be exported using the `puppet storeconfigs export` command, which will produce a tarball that can be consumed by PuppetDB. Because this command is intended only to stop nodes from failing until they have check into PuppetDB, it will only include exported resources, excluding edges and facts.
-
-NOTE: in order for this to work properly, you need to make sure you've run this command and generated the export tarball *prior* to configuring your master for PuppetDB.
-
-Once you've run this command and generated an export tarball, you should follow the instructions below to import the tarball into your PuppetDB database.
```

Comment:
I kept the link in the FAQ, imo that's probably enough since that seems to be how we navigate people to this doc in the first place.